src/features/tags/components/TagManager/components/TagsList.tsx

```diff
@@ -8,35 +9,77 @@ import TagChip from './TagChip';
 import messageIds from '../../../l10n/messageIds';
 import { useMessages } from 'core/i18n';
 
+const useStyles = makeStyles((theme) => ({
+  chip: {
+    borderColor: theme.palette.grey[500],
+    borderRadius: '1em',
+    borderWidth: '1px',
+    color: theme.palette.text.secondary,
+    cursor: 'default',
+    display: 'flex',
+    lineHeight: 'normal',
+    marginRight: '0.1em',
+    overflow: 'hidden',
+    padding: '0.2em 0.7em',
+    textOverflow: 'ellipsis',
+  },
+}));
+
 const TagsList: React.FunctionComponent<{
+  cap?: number;
   isGrouped: boolean;
   onUnassignTag?: (tag: ZetkinTag) => void;
   tags: ZetkinTag[];
-}> = ({ tags, isGrouped, onUnassignTag }) => {
+}> = ({ cap = Infinity, isGrouped, onUnassignTag, tags }) => {
+  const classes = useStyles();
   const messages = useMessages(messageIds);
 
+  const renderCappedTags = (tags: ZetkinTag[]) => {
+    const cappedTags = tags.slice(0, cap);
+    const hiddenTags = tags.slice(cap);
+    const isCapped = tags.length > cappedTags.length;
+    const tooltipTitle =
+      hiddenTags
+        .slice(0, 3)
+        .map((tag) => tag.title)
+        .join(', ') + (hiddenTags.length > 3 ? ', ...' : '');
```

Comment:
I must have turned this around in my head, somehow! Yes, it should display the number of remaining ones, let me fix that real quick.

Reply:
Alright, I think I misunderstood your question:
The reason I cap the number of titles shown in the tooltip is that otherwise it could be potentially massive. Which was the problem we tried to avoid for the hover card, so I assumed something similar here. That said, it could certainly by higher than just 3 titles.

Reply:
I ran this by Josef and Victor on the design team and after a bit of discussions the direction they want this to go is to cap at 10 visible tags instead of 3, remove the tooltip on the "+X tags" and instead have that chip link to the person's profile page (where all tags are visible). Sounds ok?